kie-dmn-feel/pom.xml

```diff
@@ -71,6 +71,26 @@
 
   <build>
     <plugins>
+    
+        <plugin>
+          <artifactId>maven-compiler-plugin</artifactId>
+          <configuration>
+            <fork>true</fork>
+            <meminitial>128m</meminitial>
+            <maxmem>512m</maxmem>
+            <showDeprecation>true</showDeprecation>
+            <showWarnings>true</showWarnings>
+            <source>1.8</source>
+            <target>1.8</target>
+            <testSource>1.8</testSource>
+            <testTarget>1.8</testTarget>
```

Comment:
You are right, I tried with the following, I just had to make sure to include the default arguments (from parent) and not just the only new `-parameters`:
```
        <plugin>
          <artifactId>maven-compiler-plugin</artifactId>
          <configuration>
            <compilerArgs>
              <arg>-Xlint:unchecked</arg>
              <arg>-parameters</arg>
            </compilerArgs>
          </configuration>
        </plugin>
```

and it worked, the "resulting POM" in eclipse was previewed as:
```
      <plugin>
        <artifactId>maven-compiler-plugin</artifactId>
        <version>3.5.1</version>
        <executions>
          <execution>
            <id>default-compile</id>
            <phase>compile</phase>
            <goals>
              <goal>compile</goal>
            </goals>
            <configuration>
              <compilerArgs>
                <arg>-Xlint:unchecked</arg>
                <arg>-parameters</arg>
              </compilerArgs>
              <showDeprecation>true</showDeprecation>
              <showWarnings>true</showWarnings>
              <source>1.8</source>
              <target>1.8</target>
              <testSource>1.8</testSource>
              <testTarget>1.8</testTarget>
            </configuration>
          </execution>
          <execution>
            <id>default-testCompile</id>
            <phase>test-compile</phase>
            <goals>
              <goal>testCompile</goal>
            </goals>
            <configuration>
              <compilerArgs>
                <arg>-Xlint:unchecked</arg>
                <arg>-parameters</arg>
              </compilerArgs>
              <showDeprecation>true</showDeprecation>
              <showWarnings>true</showWarnings>
              <source>1.8</source>
              <target>1.8</target>
              <testSource>1.8</testSource>
              <testTarget>1.8</testTarget>
            </configuration>
          </execution>
        </executions>
        <configuration>
          <compilerArgs>
            <arg>-Xlint:unchecked</arg>
            <arg>-parameters</arg>
          </compilerArgs>
          <showDeprecation>true</showDeprecation>
          <showWarnings>true</showWarnings>
          <source>1.8</source>
          <target>1.8</target>
          <testSource>1.8</testSource>
          <testTarget>1.8</testTarget>
        </configuration>
      </plugin>
```

Anyway, just spoke to Edson and I should not inspect the actual Java source code for the parameter name, but use their annotation instead; so this won't be longer necessary.

Thanks anyway @psiroky and sorry for the confusion!